mlflow/tracking/_tracking_service/registry.py

```diff
@@ -32,6 +32,6 @@ def get_store(self, store_uri=None, artifact_uri=None):
                  requirements.
         """
         from mlflow.tracking._tracking_service import utils
-        store_uri = store_uri if store_uri is not None else utils.get_tracking_uri()
+        store_uri = utils._resolve_tracking_uri(store_uri)
```

Comment:
same comment as for `mlflow/tracking/_model_registry/registry.py` on `store_uri = ""`